akka-actor/src/main/scala/akka/util/PrettyDuration.scala

```diff
@@ -1,7 +1,7 @@
 /**
  * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
  */
-package akka.testkit.metrics.reporter
+package akka.util
```

Comment:
Should we add a default argument, e.g. `def format(duration: Duration, includeNanos: Boolean = false, precision: Int = 4)`? In this example, both arguments are identical to the implementations of `PrettyPrintableDuration.pretty`